platform/ios/MGLAccountManager.m

```diff
@@ -38,15 +38,15 @@ + (instancetype) sharedInstance {
 }
 
 + (void) setAccessToken:(NSString *) accessToken {
-    [[MGLAccountManager sharedInstance] setAccessToken:accessToken];
+    [[MGLAccountManager sharedManager] setAccessToken:accessToken];
 
     // Update MGLMapboxEvents
     // NOTE: This is (likely) the initial setup of MGLMapboxEvents
-    [MGLMapboxEvents setToken:accessToken];
+    [MGLMapboxEvents sharedManager];
```

Comment:
Is there potential here for this call to be optimized away by ARC since it's not referred to strongly anywhere? Dumb idea, but curious about it.